src/ds/DLlist/createDL.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * creates a new doubly linked list head node object suitable for use in state
+ * 
+ * @param {{id:string}} config - destructured initialization object
+ * @param {string} [config.id='tree'] - id to give the DLlist, should be unique
+ * @return {{id:string, next:object, parent:object}} - DLlist head node plain data object
+ * @property {string} id - node identifier
+ * @property {?*} data - node data
+ * @property {object<{id:string, next:object}>} next - next DLlist node plain data object
+ * 
```

Comment:
Your ESDoc does not match the code you have here, and some of it is invalid syntax. [Read the ESDoc syntax docs](https://esdoc.org/manual/usage/tags.html) and make sure that the output from `npm run docs` gives you what you expect to see. Otherwise you will have confused users. 😄